DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Configuration/RegistrationPromptsController.cs

```diff
@@ -73,17 +78,46 @@ public IActionResult EditRegistrationPrompt(int promptNumber)
                 .CustomPrompts
                 .Single(cp => cp.RegistrationField.Id == promptNumber);
 
-            var data = TempData.Get<EditRegistrationPromptData>();
+            var data = new EditRegistrationPromptTempData

```

Comment:
This has been merged incorrectly. On master, `Route("Edit/Start/{promptNumber:int}")]` and `[Route("{promptNumber:int}/Edit")]` have been combined